addons/l10n_de_skr03/data/account_tax_fiscal_position_data.xml

```diff
@@ -969,23 +1081,38 @@
             <field name="children_tax_ids" eval="[(6,0,[ref('tax_ust_7_purchase_13a_auslagerung_skr03'), ref('tax_vst_7_purchase_13a_auslagerung_skr03')])]"/>
             <field name="tax_group_id" ref="tax_group_0"/>
         </record>
+        <record id="fiscal_position_domestic_skr03" model="account.fiscal.position.template">
+            <field name="sequence">1</field>
+            <field name="name">Geschäftspartner Inland</field>
+            <field name="chart_template_id" ref="l10n_de_chart_template"/>
+            <field name="auto_apply" eval="True" />
+            <field name="country_id" ref="base.de"></field>
+        </record>
         <record id="fiscal_position_non_eu_partner_service_skr03" model="account.fiscal.position.template">
+            <field name="sequence">6</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
-            <field name="name">Dienstleister Ausland</field>
+            <field name="name">Dienstleister Ausland (Nicht-EU)</field>
         </record>
         <record id="fiscal_position_non_eu_partner_skr03" model="account.fiscal.position.template">
+            <field name="sequence">5</field>
             <field name="chart_template_id" ref="l10n_de_chart_template"/>
-            <field name="name">Geschäftspartner Ausland</field>
+            <field name="name">Geschäftspartner Ausland (Nicht-EU)</field>
```

Comment:
auto_apply also needed?

Reply:
If there would be a country group similar to the group "Europe" named "Non EU Countries" with a mapping of non-eu countries into that group we could certainly auto apply this fiscal position. As it isn't available without being created by a user i would propose not to set "auto-apply" on this fiscal position. 

Maybe we should add a line:
<field name="note">Create a country group 'Non-EU Countries' in order to configure 'Auto-Apply' for this fiscal position.</field>

Reply:
You could just add nothing  (no country or country-group) in the Non-EU Countries and put it a higher sequence (so it is checked last).  The fiscal positions have a sequence that can come in handy.

Reply:
Here we also have for the partners from non-eu countries different required taxes to map for the case if we sell physical goods or services. Therefore it doesn't make sense to auto-detect the fiscal position, as we don't know if we buy or sell f.e. from the USA a service or a physical product.

Reply:
The Netherlands have a similar thing e.g., but I see the auto_apply for fiscal positions seems well configured for France.  There is a way to know it if the tax by default on the product is for a service or a physical good.

Reply:
Yes i have investigated also netherlands and france in order to benefit from auto-application of fiscal position before this commit. The only chance i personally see to get proper auto-application for fiscal position if the partner is from outside the eu would be to duplicate our "standard taxes" we usually need for germany in order to split them up from f.e. "19% VAT" to "19% VAT (for goods)" and "19% VAT (for services)" and then add both lines into the corresponding fiscal positions to map these new domestic taxes (for services) to the right different Eu / Non-Eu taxes. However this would definitely require to re-configure the taxes for service products in order to get proper results.

As a result of this new taxes we could reduce the fiscal positions and could also benefit from auto-apply feature. However users needs to be aware that they need to re-configure their service products. And finally we need duplication of 8 taxes. Furthermore it could be that some direct account -> tax mappings needs to be updated too to reflect that change.

I am open minded to prepare that change, except you point me to another alternative or better way.